library/Vanilla/Formatting/Formats/RichFormat.php

```diff
@@ -58,14 +58,18 @@ public function __construct(Quill\Parser $parser, Quill\Renderer $renderer, Quil
     /**
      * @inheritdoc
      */
-    public function renderHTML(string $content): string {
+    public function renderHTML(string $content, bool $throw = false): string {
         try {
             $operations = Quill\Parser::jsonToOperations($content);
             $blotGroups = $this->parser->parse($operations);
             return $this->renderer->render($blotGroups);
         } catch (\Throwable $e) {
```

Comment:
Throwable was specifically put here because of previous bugs in the parsing/rendering logic with certain types of garbage input. Once again we want a bug to mean a single post is is erroring instead of taking down the whole page.